main.js

```diff
@@ -1,3 +1,4 @@
+const addUnhandledErrorHandler = require('electron-unhandled');
```

Comment:
I would like us to separate built-ins (`fs`, `path`) from third-party node modules, from our own stuff. Can we move this to the node modules location?

Reply:
Sure, no problem.